### DISCUSSION TITLE
AAWireless dongle.. con auto wifi..

### DISCUSSION THREAD

**Discussion by Roscopct** [AndreyPavlenko/Fermata#521] [Announcements]
Sig. Andrey chiedo il suo aiuto..
Ho acquistato il dongle AAWireless per ovviare al problema di Android 14 e app di terze parti.. ho collegato il dispositivo via cavi all'auto.. ho collegato il telefonino al dongle.. ma purtroppo non riesco a far funzionare Android Auto sull'auto.. in precedenza collegavo il cellulare all'auto mediante wifi.. non capisco cosa devo disattivare ora utilizzando il dongle.. e passo passo i collegamenti.. chiedo il suo prezioso aiuto se potrà darmelo.. sperando infine di poter riutilizzare fermata auto..
Ringrazio..

**Comment by AndreyPavlenko** [comment]
You need to connect to AAWireless and enable the developer mode in the AAWireless settings.

**Reply by Roscopct** [reply]
Ma la connessione wifi e bluetooth tra il telefonino e l'auto devo scollegarla e dimenticarla o lascio tutto com'è??

**Reply by AndreyPavlenko** [reply]
I'm not aware of your setup, but I think you should remove the current configuration.

**Comment by Roscopct** [comment]
Inoltre le impostazioni all'interno di Android Auto le lascio come sono senza cambiare nulla?? O devo togliere connessione wifi e altro??
Ringrazio per l'aiuto..

**Reply by AndreyPavlenko** [reply]
You need to pair AA with the AAWireless device. Try resetting the AA app cache on your phone.

**Comment by Roscopct** [comment]
La mia auto.. octavia serie 4.. ha la connettività wifi.. e due prese usb.. ma a questo punto non son sicuro possa funzionare o si??

**Comment by Roscopct** [comment]
Domani riproverò..  ringrazio notte
